app/view_utils/paypal_helper.rb

```diff
@@ -20,4 +20,8 @@ def user_and_community_ready_for_payments?(user, community)
     PaypalHelper.personal_account_prepared?(user, community) &&
       PaypalHelper.community_ready_for_payments?(community)
   end
+
+  def missing_payment_info?(user, community)
```

Comment:
Naming should make it clear that the result is dependent on user having open listings, not just missing payment info.